CMakeLists.txt

```diff
@@ -745,3 +773,46 @@ if (EMSCRIPTEN)
     LINK_DEPENDS "${WABT_POST_JS};${EMSCRIPTEN_EXPORTED_JSON}"
   )
 endif ()
+
+# Create find_package configuration files
+if (WABT_INSTALL_RULES)
+  include(CMakePackageConfigHelpers)
+
+  set(WABT_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/wabt"
+      CACHE STRING "Path to wabt CMake files")
+
+  install(
+    EXPORT wabt-targets
+    DESTINATION "${WABT_INSTALL_CMAKEDIR}"
+    NAMESPACE wabt::
+    FILE wabt-targets.cmake
+    COMPONENT wabt-development
+  )
+
+  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/wabt-config.cmake.in" [[
+cmake_minimum_required(VERSION 3.8)
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/wabt-targets.cmake")
+check_required_components(wabt)
```

Comment:
Why inline this file here rather than checking it in?

Reply:
It was expedient, but I'm happy to check it in under `scripts/` or to resurrect the `cmake/` folder.

Reply:
I moved this to `scripts/`.